src/pages/Home/Home.test.tsx

```diff
@@ -35,3 +35,14 @@ test("clicking log out button logs user out", () => {
   expect(mockSetIsAuthenticated).toHaveBeenCalledWith(false);
   expect(mockSetIsAuthenticated).toHaveBeenCalledTimes(1);
 });
+
+test("clicking log out button should clear localStorage auth state", () => {
+  const authKey = "authenticated";
```

Comment:
Should I turn this into a test-only global constant? Similar to the constant declared for use in implementation code?
How do I ensure that developers won't mistakenly import the wrong one?

Reply:
you can create a test level constants if it will be used across multiple files. 
or if the constant is specific to this file, make it global for this file

Reply:
And yes at the moment that constant is used in multiple files in the test code.